src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/OppdaterVilkårsvurderingTest.kt

```diff
@@ -196,67 +197,75 @@ class OppdaterVilkårsvurderingTest {
         Assertions.assertTrue(nyInitBosattIRiketVilkår.single().resultat == Resultat.IKKE_VURDERT)
     }
 
-    // kopierOverOppfylteOgIkkeAktuelleResultaterFraForrigeBehandling
     @Test
-    fun `kopierOverOppfylteOgIkkeAktuelleResultaterFraForrigeBehandling skal kun ta med oppfylte og ikke aktuelle perioder`() {
+    fun `kopierResultaterFraForrigeBehandling skal kun ta med oppfylte og ikke aktuelle perioder såsant det ikke er ny førstegangsbehandling`() {
         val søkerAktørId = randomAktør()
-        val nyBehandling = lagBehandling()
         val forrigeBehandling = lagBehandling()
 
-        val initiellVilkårsvurdering =
-            lagVilkårsvurderingOppfylt(
-                behandling = nyBehandling,
-                personer =
-                    listOf(
-                        lagPerson(personType = PersonType.SØKER, aktør = søkerAktørId),
-                        lagPerson(personType = PersonType.BARN, aktør = randomAktør()),
+        listOf(BehandlingType.FØRSTEGANGSBEHANDLING, BehandlingType.REVURDERING).forEach { behandlingType ->
```

Comment:
Ville det ikke vært mer oversiktlig om man enten bruker @ParameterizedTest eller trekker ut koden som er felles og lager to tester i stedet for å loope gjennom to forskjellige behandlingstyper?